test/e2e/sched/install/install.go

```diff
@@ -0,0 +1,55 @@
+package install
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	nropv1alpha1 "github.com/openshift-kni/numaresources-operator/api/numaresourcesoperator/v1alpha1"
+	"github.com/openshift-kni/numaresources-operator/pkg/status"
+	e2eclient "github.com/openshift-kni/numaresources-operator/test/utils/clients"
+	"github.com/openshift-kni/numaresources-operator/test/utils/objects"
+)
+
+var _ = Describe("[Scheduler] install", func() {
+	Context("with a running cluster with all the components", func() {
+		It("should perform the scheduler deployment and verify the condition is reported as available", func() {
+			var err error
+			nroSchedObj := objects.TestNROScheduler()
+
+			By(fmt.Sprintf("creating the NRO Scheduler object: %s", nroSchedObj.Name))
+			err = e2eclient.Client.Create(context.TODO(), nroSchedObj)
+			Expect(err).NotTo(HaveOccurred())
+
+			err = e2eclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(nroSchedObj), nroSchedObj)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("checking that the condition Available=true")
+			Eventually(func() bool {
+				updatedNROObj := &nropv1alpha1.NUMAResourcesScheduler{}
+				err := e2eclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(nroSchedObj), updatedNROObj)
+				if err != nil {
+					klog.Warningf("failed to get the RTE resource: %v", err)
+					return false
+				}
+
+				cond := status.FindCondition(updatedNROObj.Status.Conditions, status.ConditionAvailable)
+				if cond == nil {
+					klog.Warningf("missing conditions in %v", updatedNROObj)
+					return false
+				}
+
+				klog.Infof("condition: %v", cond)
+
+				return cond.Status == metav1.ConditionTrue
```

Comment:
we should also add some extra testing (here or elsewhere) to test that if the condition available is true, well, the relevant pods are also running; having a condition set but no objects actually running is not that good :)

Could be done on a separate PR (it's probably the best approach actually)